sources/packages/backend/apps/api/src/route-controllers/application-offering-change-request/_tests_/e2e/application-offering-change-request.institutions.controller.createApplicationOfferingChangeRequest.e2e-spec.ts

```diff
@@ -0,0 +1,417 @@
+  const OLD_ENV = process.env;
+
+  beforeAll(async () => {
+    const { nestApplication, dataSource } = await createTestingAppModule();
+    app = nestApplication;
+    db = createE2EDataSources(dataSource);
+    // College F.
+    const { institution: collegeF } = await getAuthRelatedEntities(
+      db.dataSource,
+      InstitutionTokenTypes.CollegeFUser,
+    );
+    collegeFLocation = createFakeInstitutionLocation({ institution: collegeF });
+    await authorizeUserTokenForLocation(
+      db.dataSource,
+      InstitutionTokenTypes.CollegeFUser,
+      collegeFLocation,
+    );
+    // Institution token.
+    institutionUserToken = await getInstitutionToken(
+      InstitutionTokenTypes.CollegeFUser,
+    );
+
+    endpoint = `/institutions/location/${collegeFLocation.id}/application-offering-change-request`;
+
+    // To check the study overlap, the BYPASS_APPLICATION_SUBMIT_VALIDATIONS needs to be set false.
+    process.env = {
+      ...OLD_ENV,
+      BYPASS_APPLICATION_SUBMIT_VALIDATIONS: "false",
+    };
+  });
+
+  it("Should able to submit application offering request for an eligible application by an institution.", async () => {
+    // Arrange
+    const savedUser = await db.user.save(createFakeUser());
+    // Student has a completed application to the institution.
+    const application = await saveFakeApplicationDisbursements(
+      db.dataSource,
+      { institutionLocation: collegeFLocation },
+      {
+        applicationStatus: ApplicationStatus.Completed,
+      },
+    );
+    // Student SIN Validation.
+    application.student.sinValidation = createFakeSINValidation({
+      student: application.student,
+    });
+    application.data.howWillYouBeAttendingTheProgram =
+      OfferingIntensity.fullTime;
+
+    await db.student.save(application.student);
+    await db.application.save(application);
+
+    // New offering with full time intensity.
+    const fakeOffering = createFakeEducationProgramOffering({
+      auditUser: savedUser,
+      institutionLocation: collegeFLocation,
+    });
+    // Updating study period, that belongs to the program year.
+    fakeOffering.studyStartDate = addDays(
+      5,
+      application.programYear.startDate,
+    ).toISOString();
+    fakeOffering.studyEndDate = addDays(
+      85,
+      application.programYear.startDate,
+    ).toISOString();
+    await db.educationProgramOffering.save(fakeOffering);
+
+    const payload = {
+      applicationId: application.id,
+      offeringId: fakeOffering.id,
+      reason: "Test reason.",
+    };
+
+    let responseApplicationOfferingChangeRequest = undefined;
+    // Act/Assert
+    await request(app.getHttpServer())
+      .post(endpoint)
+      .send(payload)
+      .auth(institutionUserToken, BEARER_AUTH_TYPE)
+      .expect(HttpStatus.CREATED)
+      .expect((response) => {
+        responseApplicationOfferingChangeRequest = response.body;
+        expect(responseApplicationOfferingChangeRequest).toHaveProperty("id");
```

Comment:
IMO, We can assign the id to variable(`response.body.id`) to variable and see if it exists in db.